doc/api/child_process.md

```diff
@@ -324,6 +328,10 @@ The `child_process.spawn()` method spawns a new process using the given
 `command`, with command line arguments in `args`. If omitted, `args` defaults
 to an empty array.
 
+**Note: If `options.shell` is true, never pass unsanitised user input to
```

Comment:
A string would be true, just not `true`, so maybe its already hair-splittingly correct. How do we describe this better, though? "If options.shell is truthy"? Maybe no way to do better, and its obvious? Not sure.